spec/requests/api/v0/manifests_search_spec.rb

```diff
@@ -1,35 +1,31 @@
 require 'rails_helper'
-require_relative '../../db/chores/populator'
 require "queryparams"
 
 describe "search", elasticsearch: true do
+  before(:each) do
+    Manifest.rebuild_index
+  end
+
   describe "/search" do
     it "searches all fields by default" do
-      populate_manifests(2)
-      search_for "around+the+corner"
+      2.times.map { create(:manifest, :indexed) }
```

Comment:
nice! another shorthand for this (from FactoryGirl) is `create_list(:manifest, 2, :indexed)` (not really that much shorter, though. I am cool with either)